reinstall.sh

```diff
@@ -24,6 +24,7 @@ ${PIP} uninstall -y nemo_simple_gan
 ${PIP} install -U setuptools
 
 echo 'Installing stuff'
+# ${PIP} install numba --upgrade --ignore-installed
```

Comment:
Should we keep it?

Reply:
Oh this is for my debugging, lets remove